src/components/MyInfo/MyInfo.js

```diff
@@ -1,12 +1,26 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ShoppingBag from '../ShoppingBag/ShoppingBag';
 import './MyInfo.scss';
 
 const MyInfo = () => {
+  let [item, setItem] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/sideMenuMockData.json')
```

Comment:
- npm start를 하다보면 주소가 3000이었던게 3001로 되는 경우도 있습니다.
- 어차피 public 폴더 안에서 데이터를 받아오는 것이기 때문에 앞에 주소는 없어도 됩니다.

```suggestion
    fetch('/data/sideMenuMockData.json')
```

Reply:
넵 수정하였습니다.